3634/2/commands/check_in.rb

```diff
@@ -0,0 +1,40 @@
+require 'fileutils'
+require_relative 'save_files'
+
+module CheckInCommand
+  LONGITUDE = (53.914264..53.916233).freeze
```

Comment:
move to module/class

Reply:
you make lots of code duplication, try to inherit it from some abstract class

Reply:
I've removed code to module EventHandler (helpers/event_handler.rb) and reduce its duplication